### PR TITLE
use html2pdf to make coding the pdf easier

### PR DESCRIPTION
**Pull request by CoreyAR**
Switch to html2pdf.js so that the form can be coded in html.
The pdfmake config can be slow and difficult to work with. It als requires learning their api and experimenting with it.

By creating it in html, the development experience should be better and faster.

### CODE REVIEW

File: src/strings/pdf.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    'For both original copies: have the parent(s)/legal guardian(s) and the caregiver sign the document in front a notary or two witnesses.',
+  originalCopyCaregiver:
+    'Give one original signed copy to the caregiver to keep in a secure location.  You may also want to provide the caregiver with a copy of the child’s birth certificate, school information and doctor’s information.',
+  originalCopyYou:
+    'Keep the other original signed copy of the document for yourself in a safe place.',
+  originalCopyThirdParty:
+    "Consider giving a copy to your child’s school and a copy to your child's doctor.",
+  revoke:
+    'You may revoke this power of attorney in writing at any time.  If you revoke the power of attorney, provide a copy of the written revocation to the caregiver(s) and any agency or person who has received a copy of the power of attorney so that they are on notice that it has been revoked.',
+  instructionsEsp:
```
[the remark this answers is not in the document]
Comment (CoreyAR) [reply src/strings/pdf.js]: They are ordered in the order they appear in the document.

[the remark this answers is not in the document]
Comment (CoreyAR) [reply src/strings/pdf.js]: The only translations in this file are the instructions.

**Comment by combinatorist** [comment]
Ok, I got nvm working and set to Node 8 (like travis), but now I get the following warnings when I do `yarn install`:
```
yarn install
yarn install v1.16.0
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > react-async-component@2.0.0" has unmet peer dependency "prop-types@^15.0.0".
[4/4] Building fresh packages...
Done in 6.91s.
```

And the following error when I do `yarn start`:
```
Failed to compile

./src/components/async-download-pdf/DownloadPDF.js
Module not found: Can't resolve 'pdfmake/build/pdfmake' in '/home/qwelk/src/code-for-nashville/power-of-attorney/src/components/async-download-pdf'

This error occurred during the build time and cannot be dismissed.```

Any comments, @CoreyAR?
- doesn't travis run on linux?
- should I really need fsevents, react-async-component or async-download-pdf?

**Comment by combinatorist** [comment]
Nevermind, when I expanded, I found that travis got the [same install warnings](https://travis-ci.org/code-for-nashville/power-of-attorney/builds/597051589#L216) as me. So maybe the difference is it doesn't do a `yarn start`.

**Comment by CoreyAR** [comment]
That was my fault. Looks like the last thing I did was uninstall pdfMake and I forgot to clean up the imports/ ect.

**Comment by combinatorist** [comment]
Ok, I was wondering why we had two pdf libraries! :joy: 

Well, travis was gaslighting me because it doesn't actually do `yarn start` and it doesn't do `yarn build` until it's time to deploy to github pages. What do you think about me adding one of those commands to the configuration, @CoreyAR?

**Comment by CoreyAR** [comment]
`yarn start` isn't neccessary. Travis should do static analysis of the code and if that passes it should deploy the code. That is where `yarn build` comes into play.

I'd have to dig to be sure but I'm assuming that travis is building the app because it is currently deploying merges into master.

Looks like its failing linting

**Comment by combinatorist** [comment]
Yeah, it's doing a yarn build at deploy time, but not before we merge the
PR. So, it's all checks for the PR and (I've seen this in the past) the
deploy fails and master is not updated. It can look like the code was
merged but didn't change anything, when it fact it was merged and just not
deployed. :(

**Comment by CoreyAR** [comment]
That sounds correct to me. If you merge a failing PR into master the app
should not deploy. You will need to fix the failure and push again to
master.

**Comment by CoreyAR** [comment]
`yarn start` and `yarn build` will not cause it to fail in CI. `yarn start` is the local dev server and isn't useful in CI. `yarn build` is run when deploying.

You can see at the end of the comments the status of the branch in CI. For example, this one is currently failing.

**Comment by combinatorist** [comment]
Right, but ironically, before you made the most recent fixes, travis was passing (https://github.com/code-for-nashville/power-of-attorney/pull/122#issuecomment-542417642), which is why I was confused that I couldn't get it to start or build. But then I noticed travis doesn't build in the checks.

Now it's only failing in the `yarn lint` and `yarn test` commands (still not running build or start on the PR). https://travis-ci.org/code-for-nashville/power-of-attorney/builds/598364128?utm_source=github_status&utm_medium=notification

**Pull request merged**